src/oci-umount.c

```diff
@@ -412,6 +422,119 @@ static int parent_mntid(const char *id, char *path, const struct mount_info *mnt
 	return -1;
 }
 
+static int find_mount(const char*path, struct host_mount_info *mounts_on_host, int nr_mounts) {
+	for (int i=0; i < nr_mounts; i++) {
+		if (strcmp(mounts_on_host[i].path,path) ==0)
+			return 1;
+	}
+	return 0;
+}
+
+static int readconf(const char *id, const char *path, struct host_mount_info *mounts_on_host, int *number) {
+	_cleanup_fclose_ FILE *fp = NULL;
+	_cleanup_free_ char *line = NULL;
+	ssize_t read;
+	char *real_path;
+	size_t len = 0;
+	int nr_umounts = *number;
+	/* Parse oci-umounts.conf file, canonicalize path names and skip
+	 * paths which are not a mountpoint on host */
+	fp = fopen(path, "r");
+	if (fp == NULL) {
+		if (errno == ENOENT) {
+			pr_pwarning("%s: Config file not found: %s", id, path);
+			return 0;
+		}
```

Comment:
Now we call readconf() only if stat() returned success. That means we should never get ENOENT. So is this code still required?

Reply:
Removed